langs.ts

```diff
@@ -0,0 +1,194 @@
+/** A two-letter ISO 639-1 language code.
+ * @see https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes
+ * @see https://www.loc.gov/standards/iso639-2/php/code_list.php
+ */
+export type LanguageCode = typeof LanguageCodes[keyof typeof LanguageCodes];
+
+/**
+ * @see {@link LanguageCode}
+ */
+const LanguageCodes = {
```

Comment:
```suggestion
export const LanguageCodes = {
```
because otherwise there's no reason to have this value at runtime at all